app/cdap/components/Administration/TetheringTabContent/CdfTetheringConnections/Connections.tsx

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright © 2018 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import T from 'i18n-react';
+import { HeaderContainer, HeaderTitle, BodyContainer, NoDataText } from '../shared.styles';
+
+const PREFIX = 'features.Administration.Tethering';
+const I18NPREFIX = `${PREFIX}.Connections`;
+
+const ConnectionsContainer = styled.div``;
+
+const ConnectionsHeader = styled(HeaderContainer)`
+  background-color: ${(props) => props.theme.palette.grey[700]};
+`;
+
+const Connections = (): JSX.Element => {
```

Comment:
I think in situations like this its normally better to let typescript interpret the return type, rather than returning `JSX.Element`. How do you feel about that @njbriggs ?

Reply:
In general, I agree we don't need to specify the type in this case. I don't think it's big deal to leave it in though.